.github/workflows/hedwig.yml

```diff
@@ -0,0 +1,56 @@
+name: Test hedwig
+
+on:
+  push:
+    paths-ignore:
+    - '*.md'
+    - 'LICENSE'
+  pull_request:
+    types: [opened, repoened, synchronize]
+
+jobs:
+  native-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        rust_toolchain: [nightly, stable, 1.40.0]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
`windows-latest` is definitely a waste of time.

Reply:
There are users who build code depending on this library on Windows so at least ensuring that it builds is useful.